build/update-olm.py

```diff
@@ -17,6 +19,26 @@
         'KubeVirt', 'Virtualization', 'Template', 'Performance',
         'VirtualMachine'
     ],
+    # TODO: icon?
```

Comment:
@MarSik any suggestion about icon (I guess we can skip it for now) and links?

Reply:
To be honest.. no. Especially since the operator is mostly used for hyperconverged solution and I do not even know if we have any icon there. I never expected to expose our low level components as separate "product" in the catalog.